independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,569 @@
+                  "enum": [
+                    2,
+                    4,
+                    6,
+                    11,
+                    12,
+                    13,
+                    14,
+                    19
+                  ]
+                },
+                "path": {
+                  "title": "Path",
+                  "type": "string",
+                  "description": "The Design Manager path to the directory that contains the file being created."
+                },
+                "source": {
+                  "title": "Source",
+                  "type": "string",
+                  "description": "The source code of the file."
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/content/api/v2/templates/{template_id}": {
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Delete a template (V2)",
+        "description": "Marks the selected Template as deleted. The Template can be restored later via a POST to the restore-deleted endpoint.",
+        "operationId": "TemplatesArchive",
+        "parameters": [
+          {
+            "name": "template_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the template.",
+            "x-ms-summary": "template_id",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      },
+      "put": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Update a template (V2)",
+        "description": "Updates a template. If not all the fields are included in the body, we will only update the included fields.",
+        "operationId": "TemplatesUpdate",
+        "parameters": [
+          {
+            "name": "template_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the template.",
+            "x-ms-summary": "template_id",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "source": {
+                  "title": "Source",
+                  "type": "string",
+                  "description": "The source code of the file."
+                }
+              },
+              "required": [
+                "source"
+              ]
+            },
+            "description": "The body of the template",
+            "x-ms-summary": "body",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {

```

Comment:
Please remove `securityDefinitons` since API Properties file already has the definition.

Reply:
Done